packages/components/organisms/f-checkout/stories/Checkout.stories.js

```diff
@@ -33,7 +33,7 @@ const getBasketDineInUrl = '/get-basket-dinein.json';
 const updateCheckoutUrl = '/update-checkout.json';
 const updateCheckoutRestaurantNotTakingOrdersUrl = '/update-checkout-restaurant-not-taking-orders.json';
 const updateCheckoutAdditionalItemsRequiredUrl = '/update-checkout-additional-items-required.json';
-const updateCheckoutServerErrorUrl = '/update-checkout-server-error.json';
+const updateCheckoutAccessForbiddenUrl = '/update-checkout-403.json';
```

Comment:
On line 40 `accessForbiddenErrorUrl` should be changed to `getCheckoutAccessForbiddenUrl`

Reply:
Fair point. Let me refactor that.